CHANGELOG.md

```diff
@@ -24,20 +21,23 @@ Changes since the last release
 
 ### Known Issues
 
-## v3.10.15 \[2025-07-16\]
+## v3.10.15 \[2025-07-18\]
 
 Fixed v3.10.14 POSIX mode incompatibility error. Enhanced cvmfsexec mode 1 (mountrepo) to mount repos in
 GLIDEIN_CVMFS_REPOS and to use Apptainer images from CVMFS.
 
 ### New features / functionalities
 
+-   Add GLIDEIN_OVERLOAD_ENABLED to control partial CPU and memory overload (PR #536)
```

Comment:
Does it matter if `Add` is not changed to `Added`?

Reply:
Fixed